src/Testcontainers.Pulsar/PulsarBuilder.cs

```diff
@@ -87,6 +87,7 @@ protected override PulsarBuilder Init()
             .WithPortBinding(PulsarBrokerDataPort, true)
             .WithPortBinding(PulsarWebServicePort, true)
             .WithFunctionsWorker(false)
+            .WithCreateParameterModifier(parameterModifier => parameterModifier.User = "root")
```

Comment:
If this issue is just about permission problems, can't we store the key somewhere else? For example, `SecretKeyFilePath = "/pulsar/data/.pulsar/secret.key"` should work. This makes the change in `PulsarContainer` unnecessary too.